src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -1689,6 +1703,16 @@ void QFieldCloudProjectsModel::projectListReceived()
   QJsonDocument doc = QJsonDocument::fromJson( response );
   QJsonArray projects = doc.array();
   reload( projects );
+
+  if ( projects.size() > 0 )
+  {
+    qDebug() << "Fetch list size :" << projects.size();
+    refreshProjectsList( refreshPublicProjects );
```

Comment:
Here, use rawReply->attribute( QNetworkRequest::User + {1, 2} ) to fetch offset and whether we fetch public projects or not, then use those attributes for the refreshProjectList parameters:

```suggestion
    refreshProjectsList( isPublic, offset );
```